engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts-helper.service.js

```diff
@@ -7,28 +7,26 @@
  */
 angular.module('Bastion.content-hosts').service('ContentHostsHelper',
     function () {
-        this.getSubscriptionStatusColor = function (status) {
+        this.getHostStatusIcon = function (status) {
+            var icons;
             var colors = {
-                    'valid': 'green',
-                    'partial': 'yellow',
-                    'invalid': 'red',
-                    0: 'green',
-                    1: 'yellow',
-                    2: 'red',
-                    3: 'red'
-                };
+                'valid': 'green',
+                'partial': 'yellow',
+                'invalid': 'red',
+                0: 'green',
+                1: 'yellow',
+                2: 'red',
+                3: 'red'
```

Comment:
These colors are specifically for subscription status: https://github.com/Katello/katello/blob/master/app/models/katello/subscription_status.rb#L2-L6

So i wouldn't rename the function to general HostStatus.